src/Http/Response.php

```diff
@@ -1983,6 +1984,44 @@ public function withLength($bytes)
         return $this->withHeader('Content-Length', (string)$bytes);
     }
 
+    /**
+     * Create a new response with the Link header set.
+     *
+     * ### Examples
+     *
+     * ```
+     * $response = $response->withAddedLink('http://example.com', ['rel' => 'prev'])
+     *     ->withAddedLink(['controller' => 'Posts', 'action' => 'view'], ['rel' => 'next']);
+     * ```
+     *
+     * Will generate:
+     *
+     * ```
+     * Link: <http://example.com>; rel="prev"
+     * Link: </posts/view>; rel="next"
+     * ```
+     *
+     * @param string|array $url link-value
+     * @param array $options link-param
+     * @return static
+     */
+    public function withAddedLink($url, $options = [])
+    {
+        $url = '<' . Router::url($url) . '>';
```

Comment:
Would be preferable to only accept string URLs in this method. Right now the options to `Router::url()` cannot be used.